pjmedia/src/pjmedia/vid_port.c

```diff
@@ -1143,7 +1147,7 @@ static pj_status_t get_frame_from_buffer(pjmedia_vid_port *vp,
     pj_mutex_lock(vp->frm_mutex);
     if (vp->conv.conv)
         status = convert_frame(vp, vp->frm_buf, frame);
-    else
+    else if (frame->buf)
```

Comment:
why is this necessary?
`pjmedia_frame_copy()` will perform checking on the destination frame size.

Reply:
I revert this change 
pj_memcpy(NULL,  src, 0)  no crash